lms/djangoapps/email_marketing/models.py

```diff
@@ -0,0 +1,57 @@
+"""
+Email-marketing-related models.
+"""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from config_models.models import ConfigurationModel
+
+
+class EmailMarketingConfiguration(ConfigurationModel):
+    """ Email marketing configuration """
+
+    class Meta(object):
+        app_label = "email_marketing"
+
+    sailthru_enabled = models.BooleanField(
+        default=False,
+        help_text=_('Enable Sailthru email marketing support.')
+    )
```

Comment:
I see. It might be better to just have the SailThru configuration bundled as one config model, and have additional marketers as separate models (so that they can be enabled/disabled indpendently, and can have different groups of settings).